cardano-node/src/Cardano/Tracing/Peer.hs

```diff
@@ -6,6 +6,7 @@
 module Cardano.Tracing.Peer
   ( Peer (..)
   , getCurrentPeers
+  , ppCid
```

Comment:
This can be merged with the previous commit

Reply:
Is adding an export ok?